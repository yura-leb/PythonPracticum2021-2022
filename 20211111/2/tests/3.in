fnjjn
('a', 'a'), ('a', 'a'), ('a', 'a')
(6,0.1), (0,0), (14, 0)