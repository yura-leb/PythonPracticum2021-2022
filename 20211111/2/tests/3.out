Invalid Input
Invalid Input
0.70
